inc/customizer/options/layout_single_post.php

```diff
@@ -262,7 +262,7 @@ private function header_layout() {
 						],
 					],
 					'show_labels'           => true,
-					'live_refresh_selector' => '.nv-post-cover .nv-title-meta-wrap,.entry-header .entry-title',
+					'live_refresh_selector' => '.single-post .nv-post-cover .nv-title-meta-wrap, .entry-header .entry-title',
```

Comment:
Have a less specific selector here as previously mentioned.

Reply:
@preda-bogdan here the selectors can't be less specific. If I let them be less specific, while in customizer you could change post controls and see the changes on the single page.